### ISSUE TITLE
Delete Button is not focusable in Uploadcollection (Accessibility issue)

### ISSUE THREAD

**Issue by katapp** [SAP/ui5-webcomponents#9656]
Hi, 
in the UploadCollectionItem the Delete-Button is not focusable with the tab key. But it is accessibility standard that each button can be focused.

https://sap.github.io/ui5-webcomponents-react/?path=/docs/inputs-uploadcollection--docs#simulate-upload

is this already known?

BR Kathrin

**Comment by dimovpetar** [comment]
Hello @katapp ,

When you reach any upload collection item with delete button, you can go further with the `Tab` key, or directly use `Delete` key. Have you tried it?
[Here ](https://sap.github.io/ui5-webcomponents/play/#eyJpbmRleC5odG1sIjp7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuPCFET0NUWVBFIGh0bWw-XG48aHRtbCBsYW5nPVwiZW5cIj5cblxuXG48aGVhZD5cbiAgICBcbiAgICA8c3R5bGU-XG4gICAgICAqOm5vdCg6ZGVmaW5lZCkge1xuICAgICAgICBkaXNwbGF5OiBub25lO1xuICAgICAgfVxuICAgIDwvc3R5bGU-XG5cbiAgICA8bWV0YSBjaGFyc2V0PVwiVVRGLThcIj5cbiAgICA8bWV0YSBuYW1lPVwidmlld3BvcnRcIiBjb250ZW50PVwid2lkdGg9ZGV2aWNlLXdpZHRoLCBpbml0aWFsLXNjYWxlPTEuMFwiPlxuICAgIDx0aXRsZT5TYW1wbGU8L3RpdGxlPlxuPC9oZWFkPlxuXG48Ym9keSBzdHlsZT1cImJhY2tncm91bmQtY29sb3I6IHZhcigtLXNhcEJhY2tncm91bmRDb2xvcilcIj5cbiAgICA8IS0tIHBsYXlncm91bmQtZm9sZC1lbmQgLS0-XG5cbiAgICA8dWk1LXVwbG9hZC1jb2xsZWN0aW9uPlxuICAgICAgICA8ZGl2IHNsb3Q9XCJoZWFkZXJcIiBjbGFzcz1cImhlYWRlclwiPlxuICAgICAgICAgICAgPHVpNS10aXRsZT5BdHRhY2htZW50cyAoMik8L3VpNS10aXRsZT5cbiAgICAgICAgPC9kaXY-XG5cbiAgICAgICAgPHVpNS11cGxvYWQtY29sbGVjdGlvbi1pdGVtIGZpbGUtbmFtZT1cIkxhcHRvcEhULTEwMDAuanBnXCIgdXBsb2FkLXN0YXRlPVwiQ29tcGxldGVcIj5cbiAgICAgICAgICAgIHVwbG9hZFN0YXRlPVwiQ29tcGxldGVcIlxuICAgICAgICAgICAgPGltZyBzcmM9XCJodHRwczovL3NhcC5naXRodWIuaW8vdWk1LXdlYmNvbXBvbmVudHMvaW1hZ2VzL0hULTEwMDAuanBnXCIgc2xvdD1cInRodW1ibmFpbFwiPlxuICAgICAgICA8L3VpNS11cGxvYWQtY29sbGVjdGlvbi1pdGVtPlxuXG4gICAgICAgIDx1aTUtdXBsb2FkLWNvbGxlY3Rpb24taXRlbSBmaWxlLW5hbWU9XCJMYXB0b3AuanBnXCIgdXBsb2FkLXN0YXRlPVwiVXBsb2FkaW5nXCIgdHlwZT1cIkFjdGl2ZVwiIHByb2dyZXNzPVwiMzdcIj5cbiAgICAgICAgICAgIHVwbG9hZFN0YXRlPVwiVXBsb2FkaW5nXCJcbiAgICAgICAgICAgIDxpbWcgc3JjPVwiaHR0cHM6Ly9zYXAuZ2l0aHViLmlvL3VpNS13ZWJjb21wb25lbnRzL2ltYWdlcy9IVC0xMDAwLmpwZ1wiIHNsb3Q9XCJ0aHVtYm5haWxcIj5cbiAgICAgICAgPC91aTUtdXBsb2FkLWNvbGxlY3Rpb24taXRlbT5cblxuICAgICAgICA8dWk1LXVwbG9hZC1jb2xsZWN0aW9uLWl0ZW0gZmlsZS1uYW1lPVwibGF0ZXN0LXJlcG9ydHMucGRmXCIgdXBsb2FkLXN0YXRlPVwiRXJyb3JcIiB0eXBlPVwiQWN0aXZlXCIgcHJvZ3Jlc3M9XCI1OVwiPlxuICAgICAgICAgICAgdXBsb2FkU3RhdGU9XCJFcnJvclwiXG4gICAgICAgICAgICA8dWk1LWljb24gbmFtZT1cImRvY3VtZW50LXRleHRcIiBzbG90PVwidGh1bWJuYWlsXCI-PC91aTUtaWNvbj5cbiAgICAgICAgPC91aTUtdXBsb2FkLWNvbGxlY3Rpb24taXRlbT5cblxuICAgICAgICA8dWk1LXVwbG9hZC1jb2xsZWN0aW9uLWl0ZW0gZmlsZS1uYW1lPVwiTm90ZXMudHh0XCIgdXBsb2FkLXN0YXRlPVwiUmVhZHlcIj5cbiAgICAgICAgICAgIHVwbG9hZFN0YXRlPVwiUmVhZHlcIiAoZGVmYXVsdClcbiAgICAgICAgICAgIDx1aTUtaWNvbiBuYW1lPVwiZG9jdW1lbnQtdGV4dFwiIHNsb3Q9XCJ0aHVtYm5haWxcIj48L3VpNS1pY29uPlxuICAgICAgICA8L3VpNS11cGxvYWQtY29sbGVjdGlvbi1pdGVtPlxuICAgIDwvdWk1LXVwbG9hZC1jb2xsZWN0aW9uPlxuICAgIDwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuICAgIDxzY3JpcHQgdHlwZT1cIm1vZHVsZVwiIHNyYz1cIm1haW4uanNcIj48L3NjcmlwdD5cbjwvYm9keT5cblxuPC9odG1sPlxuPCEtLSBwbGF5Z3JvdW5kLWZvbGQtZW5kIC0tPlxuIn0sIm1haW4uanMiOnsibmFtZSI6Im1haW4uanMiLCJjb250ZW50IjoiLyogcGxheWdyb3VuZC1oaWRlICovXG5pbXBvcnQgXCIuL3BsYXlncm91bmQtc3VwcG9ydC5qc1wiO1xuLyogcGxheWdyb3VuZC1oaWRlLWVuZCAqL1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzL2Rpc3QvVGl0bGUuanNcIjtcblxuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWZpb3JpL2Rpc3QvVXBsb2FkQ29sbGVjdGlvbi5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWZpb3JpL2Rpc3QvVXBsb2FkQ29sbGVjdGlvbkl0ZW0uanNcIjtcblxuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWljb25zL2Rpc3QvZG9jdW1lbnQtdGV4dC5qc1wiO1xuXG52YXIgdXBsb2FkQ29sbGVjdGlvbiA9IGRvY3VtZW50LnF1ZXJ5U2VsZWN0b3IoXCJ1aTUtdXBsb2FkLWNvbGxlY3Rpb25cIik7XG5cbnVwbG9hZENvbGxlY3Rpb24uYWRkRXZlbnRMaXN0ZW5lcihcInJldHJ5XCIsIGUgPT4ge1xuICAgIGFsZXJ0KFwiUmV0cnkgdXBsb2FkaW5nOiBcIiArIGUudGFyZ2V0LmZpbGVOYW1lKTtcbn0pO1xudXBsb2FkQ29sbGVjdGlvbi5hZGRFdmVudExpc3RlbmVyKFwidGVybWluYXRlXCIsIGUgPT4ge1xuICAgIGFsZXJ0KFwiVGVybWluYXRlIHVwbG9hZGluZyBvZjogXCIgKyBlLnRhcmdldC5maWxlTmFtZSk7XG59KTtcblxudXBsb2FkQ29sbGVjdGlvbi5hZGRFdmVudExpc3RlbmVyKFwidWk1LWl0ZW0tZGVsZXRlXCIsIGUgPT4ge1xuICAgIHVwbG9hZENvbGxlY3Rpb24ucmVtb3ZlQ2hpbGQoZS5kZXRhaWwuaXRlbSk7XG59KTsifX0)is a working example.

Best regards,
Petar

**Comment by dimovpetar** [comment]
Hello @katapp ,

The sample you are sending is in different project - [UI5 Web Components for React](https://github.com/SAP/ui5-webcomponents-react). The upload collection which you are using is a React wrapper over the `ui5-upload-collection`, which is located in this repository. Most likely the issue comes from that UI5 Web Components for React are using ui5-webcomponents 1.24. So would you like to see this issue fixed in 1.24? 
In the future please open issues for React components in their repository.

Best regards,
Petar

**Comment by Lukas742** [comment]
Hi @dimovpetar

the agreement is that issues that only affect ui5-webcomponents should also be opened in the ui5-webcomponents repo, as we (UI5WCR) can only forward the issue in almost all cases anyway. 
If something is not clear because of our wrapper, we are of course happy to offer support, you can simply tag @MarcusNotheis or me and we will look into it.

Hi @katapp 

please always use the provided GitHub template when creating issues. This helps avoid confusion and ensures that important details, such as the version you're using, are clearly specified.

**Comment by katapp** [comment]
HI @dimovpetar could you send me the link to the react component repository for the future?
And yes sure, it would be great to see it fixed in 1.24.
best regards, Kathrin

**Comment by katapp** [comment]
Hi @dimovpetar thank you for forwarding. Where would I find the React components in their repository for future issues?

Hi @Lukas742 I'm happy to use a template, but where to find it? I opened this issue via https://sap.github.io/ui5-webcomponents/components/ > Footerlink Report issue https://github.com/SAP/ui5-webcomponents/issues/new and there is no template?

**Comment by Lukas742** [comment]
Hi @katapp 

I apologize for the miscommunication between our teams (ui5-webcomponents and ui5-webcomponents-react). Opening issues for UI5 Web Components — whether they are wrapped by our UI5 Web Components for React wrapper or not — is perfectly fine here. So, you’ve come to the right place with this issue :)

To clear up some confusion about repositories:

- UI5 Web Components are providing web components, so these components can be used with plain JS and HTML, but also with frameworks like React. It is the repository we're currently in (https://github.com/SAP/ui5-webcomponents).
- UI5 Web Components for React (UI5WCR) are wrapping the web components, so they can be more easily be used with React, since web components do work with React, but still not in the way a React developer would expect. Additionally UI5WCR is offering some more components that are not (yet) available as web component (e.g. `ObjectPage`, `AnalyticalTable` or charts). You can find the corresponding repository here: https://github.com/SAP/ui5-webcomponents-react

Regarding the issue templates:

Please always select the most appropriate issue template in this list: https://github.com/SAP/ui5-webcomponents/issues/new/choose
You can access this list by pressing the "New issue" button. 
<img width="1235" alt="image" src="https://github.com/user-attachments/assets/6c035c7a-2fa3-4c08-9e70-9f98e76ba583">


In this case, you could've chosen the "bug report" template, as the issue is regarding a bug.

**Comment by GerganaKremenska** [comment]
Hello @katapp,

Can you retest the [the web comonent for react](https://sap.github.io/ui5-webcomponents-react/v2/?path=/story/inputs-uploadcollection--simulate-upload) again. As far as i can see the issue is no longer reproducable, thinking they have updated to UI5 Web Components 2.0.  
I have tested and the issue is not available. 
Closing the issue, if it persist on your side please open
